SCClassLibrary/Common/Control/Node.sc

```diff
@@ -145,18 +147,18 @@ Node {
 			# cmd, argnodeID, parent, prev, next, isGroup, head, tail = msg;
 			// assuming its me ... if(nodeID == argnodeID)
 			Post << if(isGroup == 1, "Group:" , "Synth:") << nodeID << Char.nl
-				<< "parent  : " << parent << Char.nl
-				<< "prev : " << prev << Char.nl
-				<< "next :" << next << Char.nl;
+			<< "parent  : " << parent << Char.nl
```

Comment:
IMO this makes the code less readable, regardless of it being the IDE's indentation behavior

Reply:
ah yes, I know, it's a little annoying. I wish autoindent were a little more tolerant sometimes.  I was just too lazy to undo it, but will do it now.